app/src/main/AndroidManifest.xml

```diff
@@ -5,9 +5,6 @@
 
     <uses-sdk tools:overrideLibrary="com.google.zxing.client.android" />
 
-    <uses-permission android:name="android.permission.CAMERA" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
```

Comment:
These permissions are necessary on Android 5 and 6 for loading images in both the `ScanActivity` from gallery button or in the Photos part of `LoyaltyCardEditActivity` (see https://github.com/CatimaLoyalty/Android/issues/979#issuecomment-1283040802)

Reply:
@TheLastProject reverted